.pipeline/blubber.yaml

```diff
@@ -0,0 +1,29 @@
+version: v4
+base: docker-registry.wikimedia.org/golang:1.14-1
+lives:
+  in: /srv/service/
+runs:
+  environment:
+    APP_BASE_PATH: /srv/service
+
+variants:
+  build:
+    apt: { packages: [build-essential] }
+    copies: [local]
+    entrypoint: [make, build]
+    runs: { insecurely: false }
+  development:
+    includes: [build]
+    apt: { packages: [ca-certificates] }
+    entrypoint: [make, run]
+    runs: { environment: { APP_CONFIG_PATH: ./config.dev.yaml } }
+  test:
+    includes: [build]
+    apt: { packages: [ca-certificates] }
+    entrypoint: [make, test]
+  prep:
+    includes: [build]
+  production:
+    copies: [prep]
+    entrypoint: [make, run]
+    runs: { environment: { APP_CONFIG_PATH: ./config.prod.yaml } }
```

Comment:
i think APP_CONFIG_PATH was specific to service-runner which was part of the the node template, i dont _think_ we need it !

Reply:
@nikkhn ack. I was not sure if APP_CONFIG_PATH was the canonical way we name things in blubber pipelines (iirc we have the same convention in `similar-users`)